config/cbridge.json

```
@@ -211,11 +212,46 @@
   "linea": {
     "cBridge": "0x9B36f165baB9ebe611d491180418d8De4b8f3a1f",
     "tokensToApprove": [
+      "0xcf0f95E34F25d1BB3D9cad3Cbb2EB40dAb7c3841",
       "0x0A3BB08b3a15A19b4De82F8AcFc862606FB69A2D",
       "0x60D01EC2D5E98Ac51C8B4cF84DfCCE98D527c747",
       "0xe5D7C2a44FfDDf6b295A15c148167daaAf5Cf34f"
     ]
   },
+  "base": {
```

Biome (1.9.4)

[error] The key base was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)